### PR TITLE
Update patterns for classical Latin

### PR DESCRIPTION
**Pull request by wehro**
This merges the patterns sent to the mailing list on 3rd July 2019.

### CODE REVIEW

File: hyph-utf8/tex/generic/hyph-utf8/patterns/ptex/hyph-la-x-classic.ec.tex [hunk 1]
```diff
@@ -0,0 +1,12028 @@
...
+%     author: Claudio Beccari
+%  -
+%     date: 2019-07-03
+%     version: 2.0-EC
+%     author: Keno Wehr
+%     description: patgen-based pattern generation; improved support of
+%         compound words; do not hyphenate Roman numerals; support of v and j
+%         spellings, ae/oe ligatures, and accents
+%
+\message{Hyphenation patterns for classical Latin, v. 2.0-EC 2019-07-03}
```
Comment (reutenauer) [inline hyph-utf8/tex/generic/hyph-utf8/patterns/ptex/hyph-la-x-classic.ec.tex]: Can you please add the string
    % ==========================================
(percent, space, 42 equals signs) over this line?  It signals the end of the YAML header; without it the extraction scripts fail.

It’s also partly redundant with the message line from `loadhyph-la-x-classic.tex`.
This line is partly redundant

Comment (wehro) [reply hyph-utf8/tex/generic/hyph-utf8/patterns/ptex/hyph-la-x-classic.ec.tex]: Which file do you prefer for the message? `loadhyph-la-x-classic.tex` or the pattern file `hyph-la-x-classic.tex` itself? I can of course remove one of the messages.
But note that German also has messages in both files.

Comment (reutenauer) [reply hyph-utf8/tex/generic/hyph-utf8/patterns/ptex/hyph-la-x-classic.ec.tex]: `loadhyph`, please.  I know that German has multiple messages too, and it’s not a good idea to have redundancy there either :-)

(When I get the time I’ll work with Werner to make the format of the German header file more consistent with other languages.)

Comment (wehro) [reply hyph-utf8/tex/generic/hyph-utf8/patterns/ptex/hyph-la-x-classic.ec.tex]: OK, done.
Please have a look.

**Pull request merged**